### DISCUSSION TITLE
Viseron clock 10 hours behind

### DISCUSSION THREAD

**Discussion by love01singh** [roflcoopter/viseron#774] [Q&A] [answered]
Hi,
so the recorded video or image file name is set to default (i.e. time), but whenever there's a new file, the recorded time shows up as 10 hours behind the actual time.
For example (see picture), if the time of recoding is 12:08:40am on 20th June, the file name shows up as 14:08:40 in the 19th June log folder. It only adds next day log at 10am in the morning. 
![Screenshot](https://github.com/roflcoopter/viseron/assets/74089413/d432c172-dd43-4c46-b847-70d618dbb359)

**Comment by ItsMeRitch** [comment, marked as answer]
@love01singh 

Did you add /etc/localtime to the docker run command or as a read-only volume? 

If yes, you could also try adding your timezone (TZ) as an environment variable in the envs of your docker run command. 

eg


`-e TZ=Australia/Sydney
`

See [here](https://confluence.atlassian.com/kb/how-to-set-the-timezone-for-docker-container-976780914.html) for more info
